docs/CONTRIBUTING.md

```diff
@@ -203,6 +203,14 @@ defined by [rust-toolchain](../rust-toolchain.toml) override file. The `rustup`
 will automatically download the appropriate compiler version along with the
 necessary components.
 
+```bash
```

Comment:
I can never find the GUI doc! Why? Possibly because I am grepping just in the root `docs` subfolder - not going one directory up. Anyway I am adding a link in 0cea95eb524354447c2be7bdf2b7c8fccca62663 - so I hopefully find it next time. I kept the copy of the `bash` commands in place - they should attract my attention. Then I hopefully notice the link too.